ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml

```diff
@@ -384,6 +384,20 @@ tests:
           memory: 2Gi
       timeout: 40m0s
     workflow: servicemesh-istio-e2e-aws
+- as: istio-update-operator-2.2
+  commands: |
+    ./maistra/scripts/run-test-infra-script.sh ./tools/automator.sh \
+    -o maistra \
+    -f /creds-github/token \
+    -r istio-operator \
+    '-c make clean update-2.2-charts' \
+    '-t Automator: Update charts from maistra/istio' \
+    -m update_charts_istio
```

Comment:
You probably missed the branch `-b` argument:

`-b maistra-2.2` , `-b maistra-2.3` etc